fava/static/javascript/editor.js

```diff
@@ -179,6 +212,8 @@ $(document).ready(function() {
             event.preventDefault();
             event.stopImmediatePropagation();
 
+            CustomEvents.fire(document, "LiveReloadShutDown");
```

Comment:
@corani A simple `$(document).trigger('LiveReloadShutDown');` does the trick for me as well - no `CustomEvents`-code needed.

Can you try if this works for you as well?